docs: add troubleshooting for default image endpoints DOC-1661 (#5717)
spectrocloud/librarium: docs/docs-content/troubleshooting/enterprise-install.md

   ```bash hideClipboard
   fs.file-max = 1000000
   ```

> ## Scenario - MAAS and VMware vSphere clusters fail image resolution in non-airgap environments

Reviewer: ```suggestion
## Scenario - MAAS and VMware vSphere Clusters Fail Image Resolution in Non-Airgap Environments
```

In Palette or VerteX non-airgap installations with version 4.2.13 or later, MAAS and VMware vSphere clusters may fail to
provision due to image resolution errors. These environments have incorrectly configured default image endpoints. To
resolve this issue, you must manually set these endpoints.

### Debug Steps

1. Open a terminal with connectivity to your self-hosted environment.

> 2. Execute the following command to save base URL of your Palette instance API to the `BASE_URL` environment value. Add

Reviewer 2: ```suggestion
2. Execute the following command to save the base URL of your Palette instance API to the `BASE_URL` environment value. Add
```

   your correct URL in place of `REPLACE_ME`.

   ```shell
   export BASE_URL="REPLACE ME"
   ```

3. Use the following command to log in to the Palette System API by using the `/v1/auth/syslogin` endpoint. Ensure you
   replace the credentials below with your system console credentials.

   ```bash
   curl --location '$BASE_URL/v1/auth/syslogin' \
   --header 'Content-Type: application/json' \
   --data '{
     "password": "**********",
     "username": "**********"
   }'
   ```

   Output

   ```json hideClipboard
   {
     "Authorization": "**********.",
     "IsPasswordReset": true
   }
   ```

> 4. Using the output you received, copy the authorization value to your clipboard and assign it to an environment

Reviewer: ```suggestion
4. Copy the authorization token to your clipboard and assign it to an environment
```
maybe? to simplify

>    variable. Replace the authorization value below with the value from the output.

Reviewer: ```suggestion
   variable. Replace the placeholder below with the value from the output.
```
maybe?

   ```shell hideClipboard

>    TOKEN=**********

Reviewer 2: ```suggestion
   export TOKEN=**********
```

Reviewer 2: Needs the `export` no?

   ```

5. Execute the following command to set the MAAS image endpoint to `https://maasgoldenimage.s3.amazonaws.com`. Replace
   the `caCert` value below with the Certificate Authority (CA) certificate for your self-hosted environment.

   ```shell
   curl --request PUT '$BASE_URL/v1/system/config/maas/image' \
   --header 'Authorization: $TOKEN' \
   --header 'Content-Type: application/json' \
   --data '{
      "spec": {
         "imagesHostEndpoint": "https://maasgoldenimage.s3.amazonaws.com",
         "insecureSkipVerify": false,
         "caCert": "**********"
      }
   }'
   ```

6. Execute the following command to set the VMware vSphere image endpoint to
   `https://vmwaregoldenimage.s3.amazonaws.com`. Replace the `caCert` value below with the Certificate Authority (CA)
   certificate for your self-hosted environment.

   ```shell
   curl --request PUT '$BASE_URL/v1/system/config/vsphere/image' \
   --header 'Authorization: $TOKEN' \
   --header 'Content-Type: application/json' \
   --data '{
      "spec": {
         "imagesHostEndpoint": "https://vmwaregoldenimage.s3.amazonaws.com",
         "insecureSkipVerify": false,
         "caCert": "**********"
      }
   }'

>    ```

Reviewer: Should we add a closing sentence? Something saying that now that the endpoints are updated, we will be able to create clusters

Author: Added.
